axelor-account/src/main/java/com/axelor/apps/account/service/PaymentModeControlServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.axelor.apps.account.service;
+
+import com.axelor.apps.account.db.PaymentMode;
+import com.axelor.apps.account.db.repo.MoveRepository;
+import com.google.inject.Inject;
+import java.util.Objects;
+
+public class PaymentModeControlServiceImpl implements PaymentModeControlService {
+
+  protected MoveRepository moveRepository;
+
+  @Inject
+  public PaymentModeControlServiceImpl(MoveRepository moveRepository) {
+
+    this.moveRepository = moveRepository;
+  }
+
+  @Override
+  public Boolean isInMove(PaymentMode paymentMode) {
```

Comment:
We should return an unboxed `boolean` here, so we can be sure that it cannot be null.